### ISSUE TITLE
How many epochs are needed for loss convergence?

### ISSUE THREAD

**Issue by izuna385** [MurtyShikhar/Hierarchical-Typing#3]
Hi,

For `dataset=umls`, how many epochs are needed for loss convergence?
I can't find out epochs for experiment in paper and code.

The default ep in uploaded code is `2` , so I first run it 150 epoch.
If you could know rough estimate of epochs for loss convergence, I would be grateful.

Thanks! 

The followings are my params for umls.

```
('dataset', 'umls')
('model_name', 'entity_linker')
('dropout', 0.5)
('train', 1)
('bag_size', 10)
('encoder', 'position_cnn')
('struct_weight', 0.5)
('typing_weight', 0)
('multilabel_linking', 0)
('parent_sample_size', 100)
('bilinear_l2', 0.0005)
('bilinear_bias', 0)
('complex', 0)
('asymmetric', 1)
('learn_graph', 0)
('priors', 1)
('base_dir', './dataset_dir')
('lr', 0.001)
('beta1', 0.9)
('beta2', 0.999)
('epsilon', 1e-08)
('weight_decay', 0.0)
('save_model', 1)
('clip_val', 5)
('embedding_dim', 300)
('hidden_dim', 150)
('num_epochs', 150)
('kernel_width', 5)
('batch_size', 256)
('take_frac', 1.0)
('use_transitive', 1)
('features', 0)
('distance_func', 'dot')
('feature_dim_size', 0)
size of training data: 199977.0000
INFO:Entity Linker:
Number of words in vocab: 2195877
```
